src/app/components/RouterTabs/index.tsx

```diff
@@ -29,18 +28,18 @@ export function RouterTabs<Context>({ tabs, context }: RouterTabsProps<Context>)
 
   return (
     <>
-      <Tabs value={targetTab?.to} variant="scrollable" scrollButtons={false}>
-        {tabs
-          .filter(tab => tab === targetTab || tab.visible !== false)
-          .map(tab => (
-            <Tab
-              key={tab.to}
-              component={NonScrollingRouterLink}
-              value={tab.to}
-              label={tab.label}
-              to={tab.to}
-            />
-          ))}
+      <Tabs value={targetTab?.to}>
```

Comment:
Future issue / UX regression: on mobile if there's not enough space https://explorer.dev.oasis.io/mainnet/sapphire/address/0x65524539A0Db2dB286b3a05E4E191eA810295CE2

![Recording 2025-08-20 at 21 03 32](https://github.com/user-attachments/assets/d4b9b3be-ac8d-4ac9-bf31-eb3e8060e426)

Before: touch-scrollable
After: Transfers are offscreen on left, Code is offscreen on right

cc @juresobocan @donouwens